src/wrapped_radix_engine_toolkit/high_level/sargon_specific_types/assets_transfers/per_recipient/per_recipient_fungible_transfer.rs

```diff
@@ -0,0 +1,83 @@
+use crate::prelude::*;
+
+impl PerRecipientFungibleTransfer {
```

Comment:
The `PerRecipientFungibleTransfer ` type is declared with macro in `transfer_types.rs`